cristin-project/src/test/java/no/unit/nva/cristin/projects/common/CristinQueryTest.java

```diff
@@ -17,6 +17,28 @@ public class CristinQueryTest {
     private static final String GET_ONE_CRISTIN_PROJECT_EXAMPLE_URI =
         "https://api.cristin-test.uio.no/v2/projects/1234?lang=nb";
 
+    private static final String SAMPLE_SORT = "start_date";
+    private static final String SAMPLE_INSTITUTION_ID = "uib";
+    private static final String SAMPLE_PROJECT_MANAGER = "st";
+    private static final String SAMPLE_PARTICIPANT = "St";
```

Comment:
very short names

Reply:
In documentation it says you can just use "part of the name". I used those because they provided enough results in real cristin queries so I can check real combination of sample params. But if want them longer, easy to fix

Reply:
Just keep them as they are